.github/workflows/pytest-core-mpi.yml

```diff
@@ -17,11 +17,12 @@ on:
 jobs:
   build:
     name: pytest-mpi
-    runs-on: self-hosted
+    runs-on: [self-hosted, mpi]
 
     env:
-      DEVITO_ARCH: "gcc-7"
+      DEVITO_MPI: "1"
```

Comment:
Oh yeah no this need to go, pytest expect `DEVITO_MPI=0` and then runs it internally